biocypher_metta/adapters/forge2_tf_adapter.py

```diff
@@ -8,7 +8,7 @@
 
 class Forge2TFAdapter(Adapter):
```

Comment:
Looks like this adapter isn't implemented. Either implement the methods or maybe remove it from this PR and you can implement it in a future PR

Reply:
Alright, I'll implement it in a separate PR later on. For now, let's go ahead and remove it from this one.